src/DynamoCoreWpf/ViewModels/Preview/WatchViewModel.cs

```diff
@@ -249,7 +251,7 @@ private static string GetStringFromObject(object obj)
                 case TypeCode.Boolean:
                     return ObjectToLabelString(obj);
                 case TypeCode.Double:
-                    return ((double)obj).ToString(numberFormat, CultureInfo.InvariantCulture);
+                    return ((double)obj).ToString(ProtoCore.Mirror.MirrorData.PrecisionFormat, CultureInfo.InvariantCulture);
```

Comment:
Again I would comment this change for the time being until https://jira.autodesk.com/browse/DYN-5101 is done. Add a TODO.